### PR TITLE
Fix typos.

### PR DESCRIPTION
**Pull request by Deledrius**
- Fixes misspellings and a few errant words found in the Foundations, Intermediate, and Advanced content.
- Corrects inconsistent capitalization of GitHub terms.

### CODE REVIEW

File: advanced/index.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ Cutting and creating releases on the command line and on GitHub.
     * [Git-Flow: A Successful Branching Model](http://nvie.com/posts/a-successful-git-branching-model/)
     * [Git-Flow Source](https://github.com/nvie/gitflow)
     * Too many levels?
-    * GH prefers Simplest thing that works.
+    * GH prefers "Simplest thing that works".
```
Comment (loranallensmith) [inline advanced/index.md]: Might be good to either move this period inside the quote or drop it off entirely to remain consistent with other lists.

Comment (brntbeer) [inline advanced/index.md]: Im a fan of no periods at the end of lists personally

Comment (Deledrius) [inline advanced/index.md]: Fixed. :smile:

**Comment by matthewmccullough** [comment]
A solid set of tune-ups. Thank you @Deledrius for the edits and @loranallensmith for the inputs.

**Pull request merged**